app/controllers/rubcop_test_controller.rb

```diff
@@ -0,0 +1,17 @@
+class RubocopController  < ApplicationController
```

Comment:
__[rubocop]__
* Missing top-level class documentation comment.
* Operator `<` should be surrounded with a single space.